boards/nxp/fmuk66-v3/src/spi.cpp

```diff
@@ -131,6 +131,7 @@ __EXPORT void board_spi_reset(int ms, int bus_mask)
 			for (int i = 0; i < SPI_BUS_MAX_DEVICES; ++i) {
 				if (px4_spi_buses[bus].devices[i].cs_gpio != 0) {
 					kinetis_pinconfig(px4_spi_buses[bus].devices[i].cs_gpio);
+					kinetis_gpiowrite(px4_spi_buses[bus].devices[i].cs_gpio, 1);
```

Comment:
The default of of a CS GPIO had better have GPIO_OUTPUT_SET (GPIO_OUTPUT_ONE) so Why is this here?

Reply:
I was being paranoid :) Indeed, it is, and I removed these.